app/assets/javascripts/incidents.subquestions.js

```diff
@@ -75,4 +78,19 @@ $(function(){
     $('#multiple-agencies-info-text').toggle($('#screener_multiple_agencies_true').is(':checked'));
     $('#shots-fired-info-text').toggle($('#screener_shots_fired_true').is(':checked'));
   }).change();
+
+
+  function medicalAid() {
```

Comment:
There's no need to wrap this code in a function, since you just immediately invoke it on load. Or am I missing something?